src/flagStore.ts

```diff
@@ -64,13 +65,37 @@ export class FlagStore {
 			const ldConfig = this.ldConfig();
 			const ldClient = await LaunchDarkly.init(sdkKey, ldConfig).waitForInitialization();
 			this.resolveLDClient(ldClient);
+			this.on('update', async flag => {
+				try {
+					this.storeUpdates.fire(flag.key);
+				} catch (err) {
+					console.error('Failed to update LaunchDarkly flag store.', err);
+				}
+			});
+			if (this.config.refreshRate) {
+				if (this.config.validateCron(this.config.refreshRate)) {
+					this.cron(this.config.refreshRate);
+				} else {
+					window.showErrorMessage(
+						`Invalid cron expression: '${this.config.refreshRate}' needs to be 5 field cron format.`,
+					);
+				}
+			}
 		} catch (err) {
 			this.rejectLDClient();
 			console.error(err);
 		}
 	}
 
-	async on(event: string, cb: FlagUpdateCallback) {
+	private async cron(exp: number) {
```

Comment:
Let's give this a name that might give the reader a better idea of what it does. Maybe something like `startGlobalFlagUpdateTask(interval: number)`